apps/android/README.md

```diff
@@ -5,19 +5,31 @@
 ## Launch `FluentUI Tester` app on Android
 
 Prereqs:
+
```

Comment:
What do the pluses do?

Reply:
I think that's just a new line. I didn't add a plus